index.html

```diff
@@ -14,11 +14,11 @@
         <div id="header" name="header">
 
             <div id="upperHeader" name="upperHeader">
-                <img src="../Gatham/assets/Logo2.png" id="mainLogo" name="mainLogo">
+                <img src="../GothamAutoRenov/assets/Logo2.png" id="mainLogo" name="mainLogo">
```

Comment:
@ramindev61 all addresses inside code must be relative to the root folder of repository, in this way if you copy and paste, whole folder of repository or change the root folder name then the code will still works.
for example for this one you can use this relative address :
"./assets/Logo2.png"

Reply:
done